ha/core/config/config_manager.py

```diff
@@ -84,6 +84,13 @@ def load_filter_rules():
         """
         ConfigManager._safe_load(const.ALERT_FILTER_INDEX, f"json://{const.ALERT_FILTER_RULES_FILE}")
 
+    @staticmethod
+    def load_alert_events_rules():
+        """
+        Loads alert filter rules.
+        """
+        ConfigManager._safe_load(const.ALERT_FILTER_INDEX, f"json://{const.ALERT_FILTER_RULES_FILE}")
```

Comment:
This load should use ALERT_EVENT_INDEX and ALERT_EVENT_RULES_FILE  right ?

Reply:
Agree, fixed.